kernel/ttydev.c

```diff
@@ -322,10 +322,19 @@ static int _td_get_events(myst_ttydev_t* ttydev, myst_tty_t* tty)
     if (!ttydev || !_valid_tty(tty))
         ERAISE(-EINVAL);
 
-    if (tty->fd == STDOUT_FILENO)
-        ret |= POLLOUT;
-    else
-        ret = -ENOTSUP;
+    switch (tty->fd)
+    {
+        case STDIN_FILENO:
+            ret |= POLLIN;
```

Comment:
Does Mystikos support read from stdin? BTW, myst_tcall_read_console() implementation proceeds if the fd is stdout or stderr, and return error if fd is stdin. Is that intended?

Reply:
I'm sorry, I should have marked this one as experimental.

I think there is a security concern so we should abandon this PR.